canonical/store-components
# Add filters component

## Done
Added a filters component

## QA
- Go to https://store-components-15.demos.haus/?path=/story/mainpage--default-story
- Check that there are filters and they work as expected

## Issue
Fixes:
- https://warthogs.atlassian.net/browse/WD-1368
- https://warthogs.atlassian.net/browse/WD-1618
- https://warthogs.atlassian.net/browse/WD-1619
